### DISCUSSION TITLE
Broken picture and keep getting error while entering github hosted sites

### DISCUSSION THREAD

**Discussion by JayedRafiProjects** [community/community#11917] [Other Feature Feedback, Questions, & Ideas]
The sites are not loading and also my profile picture is also not visible. It's just showing me a broken image and link ...

**Comment by LoginAGo** [comment]
I'm getting the same erros. See https://github.com/github/feedback/discussions/11915

**Reply by JayedRafiProjects** [reply]
It's working now. Issue fixed

**Reply by LoginAGo** [reply]
Good. I forgot to update my question...
